tests/sentry/incidents/subscription_processor/test_subscription_processor_workflow_engine.py

```diff
@@ -604,7 +602,6 @@ def test_seer_call_dual_processing__critical(self, mock_seer_request: MagicMock)
     @with_feature("organizations:anomaly-detection-alerts")
     @with_feature("organizations:anomaly-detection-rollout")
     @with_feature("organizations:workflow-engine-metric-alert-processing")
-    @skip("not enabled yet")
```

Comment:
Unfortunately I don't think it's possible to write a test for the case we hit - we could handle a `JSONDecodeError`, but we definitely want to fail loudly here and know that we're not sending the data Seer expects and anomaly detection isn't working.